src/components/FaceScreen.tsx

```diff
@@ -58,7 +59,7 @@ const FaceScreen: React.FC<FaceScreenProps> = () => {
           thinking as they'll be building web interfaces with accessibility in
           mind. They should also be excited to learn, as the world of Front-End
           Development keeps evolving.
-        </Typography>
+        </CustomisedTypography>
```

Comment:
Better to use sense full naming for custom components

Reply:
fixed